src/m_menu.c

```diff
@@ -2968,6 +2968,7 @@ static void M_KeyBindings(int choice)
   default_verify = false;
   setup_gather = false;
   mult_screens_index = M_GetMultScreenIndex(keys_settings);
+  set_tab_on = mult_screens_index;
```

Comment:
Is `set_tab_on` redundant with `mult_screens_index`?

Reply:
`set_tab_on` is the index of the tab the mouse cursor is on ("highlighted"), `mult_screens_index` changes when the tab is switched.

Yes, we currently have `setup_screens` and tabs which duplicate the same functionality, but I didn't want to touch old functions like `M_ResetDefault`/`M_InitDefaults` etc. We should simplify this in the future.